src/icrar/leap-accelerate-cli/tests/E2EPerformanceTests.cc

```diff
@@ -98,7 +98,9 @@ namespace icrar
             }
             else if(impl == ComputeImplementation::cuda)
             {
+#ifdef CUDA_ENABLED
```

Comment:
Can use runtime function instead of compilation-time macro, as per the suggestion in the global review comment.

Reply:
the signature `cuda::Calibrate()` does not exist if the macro is disabled. If I took the guards away from the header it will then produce a link error unless I go through and stub all public cuda functions in the project.

Reply:
True, hadn't spotted that. On the other hand now that you have `icrar::Calibrate` you could invoke that here instead of repeating the `if/else if` logic.